provisioning_client/samples/readme.md

```diff
@@ -0,0 +1,19 @@
+# Samples for Azure IoT Provisioning SDK
+
+## List of samples
+
+- custom_hsm_examples
+
+  - This example shows how to create a custom HSM to retrieve the secruity information needed for the provisioning SDK to communication the the DPS Service.
+
+- iothub_client_sample_hsm
+
+  - This sample show how to use the IoThub Device Client with the device authentication system without using provisioning.
+
+- prov_dev_client_ll_sample
+
+  - This sample uses the non-threaded SDK to connect the provisioning client and get the IoThub information and then connects to the iothub.  This example would be for devices that require greater control over communications by calling DoWork to control the message pump.
+
+- prov_dev_client_sample
```

Comment:
If I'm just getting started and don't have hardware yet (but I'm a C developer), which one of these samples do you recommend using? Or is it obvious to a real C developer?